nautobot/core/api/views.py

```diff
@@ -418,7 +418,7 @@ class ModelViewSet(
     BulkDestroyModelMixin,
     ModelViewSetMixin,
     ModelViewSet_,
-    MyFormSchemaViewSetMixin,
+    # MyFormSchemaViewSetMixin,  # TODO: This can/should be limited to /api/ui/ schemas
```

Comment:
This was causing issues with schema output. I think this should be opt-in via either a header or something else but is a breaking change to the APIs for what amounts to UI data.